typings/internalBinding/wasi.d.ts

```diff
@@ -0,0 +1,14 @@
+declare namespace InternalWASIBinding {
+    type EnvStr = `${string}=${string}`
```

Comment:
Can you change indentation to 2?

Reply:
Oh sorry, I missed the `.editorconfig`. fixed!